src/csm_testing/tests/sat_functional/hwinv/test_hwinv.py

```diff
@@ -0,0 +1,630 @@
+#
+# MIT License
+#
+# (C) Copyright 2025 Hewlett Packard Enterprise Development LP
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+"""
+Simple tests for the functionality of the `sat hwinv` command.
+"""
+import shlex
+import subprocess
+import unittest
+from typing import List
+
+HWINV_NODE_HEADER = [
+    'xname', 'FRUID', 'Manufacturer', 'Model', 'Part Number', 'SKU', 'Serial Number',
+    'Cabinet Type', 'Memory Type', 'Memory Device Type', 'Memory Manufacturer',
+    'Memory Size (GiB)', 'Memory Module Count', 'Processor Count', 'Processor Manufacturer',
+    'Processor Model', 'Accelerator Count', 'Accelerator Riser Count', 'HSN NIC Count',
+    'Drive Count', 'Total Drive Capacity (GiB)', 'BIOS Version', 'Memory Model'
+]
+HWINV_CHASSIS_HEADER = [
+    'xname', 'FRUID', 'Manufacturer', 'Model', 'Part Number', 'Serial Number', 'SKU'
```

Comment:
It looks like SKU always comes before "Serial Number":

```suggestion
    'xname', 'FRUID', 'Manufacturer', 'Model', 'Part Number', 'SKU', 'Serial Number'
```

Reply:
I won't comment on the other instances of this, but it appears to be the same throughout.